x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -96,15 +97,9 @@ def database_path
   protected
   # return a valid database path or default database path
   def patch_database_path(database_path)
-    unless file_exist?(database_path)
-      database_path = get_file_path("GeoLite2-#{@database_type}.mmdb")
-
-      unless file_exist?(database_path)
-        raise "You must specify 'database => ...' in your geoip filter (I looked for '#{database_path}')"
-      end
-    end
-
-    database_path
+    return database_path if file_exist?(database_path)
+    return database_path if database_path = get_file_path("GeoLite2-#{@database_type}.mmdb") and file_exist?(database_path)
```

Comment:
As I understand  this `if statement` checks for default database path, so be more specific could help readability:
```suggestion
    default_database_path = get_file_path("GeoLite2-#{@database_type}.mmdb")
    return default_database_path if default_database_path and file_exist?(default_database_path)
```

Maybe the `not nil` check of `default_database_path` could be embedded into `file_exists?`